core/src/main/java/io/aiven/klaw/helpers/DefaultAttributeConverter.java

```diff
@@ -37,7 +37,7 @@ default X convertToEntityAttribute(String value) {
   default X convertToEntityAttribute(String value, X defaultValue) {
     X res = defaultValue;
     try {
-      if (value != null) {
+      if (value != null && !value.equals("")) {
```

Comment:
Strings.isEmpty(value)

that will check if its null or if the string has a length of 0 (empty)